.github/workflows/cmake.yml

```
@@ -175,6 +176,7 @@
             asan-options: detect_leaks=0
             packages: qemu-user crossbuild-essential-armhf
             gcov-exec: arm-linux-gnueabihf-gcov
+            cmake-args: -DWITH_BENCHMARKS=ON
```

actionlint (1.7.4)

key "cmake-args" is duplicated in element in "include" section. previously defined at line:175,col:13. note that this key is case insensitive

(syntax-check)